sections/persons.md

```diff
@@ -26,6 +26,8 @@
 | default_role        | Integer, ID of default role                                                            |
 | cost                | Decimal, cost from the current cost period                                             |
 | language            | String {"SPANISH", "DANISH", "FRENCH", "ENGLISH_EU", "ENGLISH_UK", "ENGLISH_US"}       |
+| start_date          | String                                                                                 |
+| end_date            | String                                                                                 |
```

Comment:
Can you put "ISO 8601" in the Description/format column here?

Reply:
added it